src/MobileDevice/widget/MobileDevice.xml

```diff
@@ -0,0 +1,66 @@
+    <properties>
+        <property key="deviceEntity" type="entity" required="false">
```

Comment:
It is needed when the device information is being passed back to the microflow to be executed. so that the user can get first hand the object that has just be update...Well i can remove it but i think it makes more sense to have it in that view